internal/sdkv2provider/schema_cloudflare_notification_policy.go

```diff
@@ -34,6 +34,7 @@ func resourceCloudflareNotificationPolicySchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Required: true,
 			ValidateFunc: validation.StringInSlice([]string{
+				"advanced_http_alert_error",
```

Comment:
this also needs to be added to the `Description` field below to automatically update the documentation - https://github.com/cloudflare/terraform-provider-cloudflare/pull/2789/files#diff-1d41b68abb4996509e6bdcd228d7140d6dc187ef2b020c82244988361e6a54aeR87